tests/integration/helpers/element-test.js

```diff
@@ -220,7 +230,7 @@ module('Integration | Helper | element', function(hooks) {
 
     await settled();
 
-    assert.equal(this.element.innerHTML.trim(), 'Test');
+    assert.strictEqual(this.element.innerHTML.trim(), 'Test');
```

Comment:
I think the failing test in ember-classic is due to `'template-only-glimmer-components': false`, thus `<ElementReceiver>` has an additional wrapper div. 

We could do this instead, which would hopefully fix the problem:

```suggestion
    assert.strictEqual(this.element.innerText.trim(), 'Test');
```